tests/integration/bond_test.py

```diff
@@ -89,3 +92,30 @@ def test_add_bond_without_slaves():
 
     netapplier.apply(copy.deepcopy(desired_bond_state))
     assertlib.assert_state(desired_bond_state)
+
+
+def test_add_bond_with_slaves_and_ipv4():
+    desired_bond_state = {
+                INTERFACES: [
+                    {
+                        'name': 'bond99',
+                        'type': 'bond',
+                        'state': 'up',
+                        'ipv4': {
+                            'enabled': True,
+                            'address': [
+                                {'ip': '192.168.122.250', 'prefix-length': 24}
+                            ]
+                        },
+                        'link-aggregation': {
+                            'mode': 'balance-rr',
+                            'slaves': ['eth1', 'eth2'],
+                            'options':
+                                {'miimon': '140'}
+                        },
+                    }
+                ]
+            }
+
+    netapplier.apply(copy.deepcopy(desired_bond_state))
```

Comment:
Please separate between the tested action and the assert with a new-line for readability.